docs/platforms/elixir/integrations/plug_and_phoenix/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Plug and Phoenix
+description: "Learn how to capture errors and their context in Plug pipelines and Phoenix applications."
+sidebar_order: 10
+---
+
+You can capture errors in [Plug](https://github.com/elixir-plug/plug) and [Phoenix](https://phoenixframework.org) applications with `Sentry.PlugContext` and `Sentry.PlugCapture`. `Sentry.PlugContext` adds contextual metadata from the current request, which is then included in errors that are captured and reported by `Sentry.PlugCapture`.
+
+## Phoenix Configuration
+
+If you are using Phoenix:
+
+  1. Add `Sentry.PlugCapture` above the `use Phoenix.Endpoint` line in your endpoint file.
+  1. Add `Sentry.PlugContext` below `Plug.Parsers`.
+
+```diff
```

Comment:
```suggestion
```Elixir
```
Was there a reason to say diff instead of elixir for the code block?

Reply:
This was originally in the docs for the SDK (on hex.pm), and I left it mostly untouched. I really do prefer the diff since it shows only the lines you need to add and where, while using Elixir syntax highlighting makes it harder to understand where the changes are IMO.